src/core/utils.js

```diff
@@ -843,3 +843,7 @@ export function paramToValue(param, paramValues) {
 
   return values[0]
 }
+
+export function xclass(...args) {
```

Comment:
**Change request**: use [`classnames`](https://www.npmjs.com/package/classnames) instead of keeping `xclass` around. At a glance, looks like we can switch out the invocations without changing anything else.

(I know this PR didn't introduce the function, but since we're cleaning up `layout-utils`, might as well fix this.)

Reply:
No problem, removed 👍

Reply:
And nope, I didn't make any changes beside moving things around. I didn't want to go outside of the scope of this ticket. So the main files that need attention are  src/core/components/layout-utils/drop-down.jsx`, `src/core/components/schemes.jsx` and `src/style/_dropdown.scss`.